day1/Vagrantfile

```diff
@@ -2,32 +2,138 @@
 # vi: set ft=ruby :
 
 Vagrant.configure("2") do |config|
-  config.vm.define "zabbix_server" do |server|
-    server.vm.hostname = "server"
-    server.vm.box = "sbeliakou/centos-7.3-x86_64-minimal"
-    server.vm.network "forwarded_port", guest: 80, host: 8080
-    server.vm.network "private_network", ip: "192.168.56.10"
-    server.vm.provider "virtualbox" do |vb|
-      vb.name = "Zabbix Server"
-      vb.memory = "1024"
+
+ config.vm.define "zabbix" do |zb|
+    zb.vm.box = "sbeliakou/centos-7.3-x86_64-minimal"
+    zb.vm.network "private_network", ip: "111.111.11.11"
+    zb.vm.network "public_network", bridge: "eno1"
```

Comment:
The idea was to create more screens from home via vpn-connection. In fact I failed to did so. This part was left for the case I make It work fine.